src/components/BookPage.js

```diff
@@ -14,7 +13,7 @@ function BookPage() {
     dispatch(booksFromAPI());
   }, []);
   return (
-    <>
+    <div style={{ border: 'solid 3px #e8e8e8', height: '100vh' }}>
       <Nav />
       <div className={style.body}>
         <div className={style.action}>
```

Comment:
- [OPTIONAL] Great work implementing the changes. It would be nice if you removed this style rule as it doesn't conform with the application's layout:

```CSS
border: 'solid 3px #e8e8e8'
```

If you notice in the original design, it is the header that contains the navigation link and logo that has a border-bottom with the colour. Also, it would be nice to conform with the padding and background colour of the BookPage component. 👍 😎 

### Zeplin Design

![image](https://github.com/mohamedgamaleldin999999/The-bookstore/assets/77106962/e14040b4-467c-4474-b7fd-4fcf174f10cf)